packages/build-utils/src/types.ts

```diff
@@ -411,7 +410,11 @@ export interface BuildResultBuildOutput {
   buildOutputPath: string;
 }
 
-export type Cron = string;
+export interface Cron {
+  path: string;
+  schedule: string;
```

Comment:
Do we also need a `region` property here? A function might be deployed to multiple regions so its not clear if all regions are invoked or just one?

Reply:
The requests for Cron Jobs are made from one region at the moment over public internet, so it doesn't matter in this case.